lambdas/checklist/serverless.yml

```diff
@@ -40,9 +41,7 @@ functions:
       command:
         - evaluate.handler
     events:
-      - http:
-          path: /checklists/evaluate
-          method: POST
+      - sqs: ${ssm:/terraform/${self:provider.stage}/sqs/criteria_evaluate_queue_arn}
```

Comment:
I will rename this to:
```
${ssm:/terraform/${self:provider.stage}/sqs/criteria_evaluation_processor_queue_arn}
```

Reply:
maybe checklist_evaluation_processor_queue_ar is better?